lib/rubocop/cop/lint/redundant_safe_navigation.rb

```diff
@@ -7,6 +7,10 @@ module Lint
       # `instance_of?`, `kind_of?`, `is_a?`, `eql?`, `respond_to?`, and `equal?` methods
       # are checked by default. These are customizable with `AllowedMethods` option.
       #
+      # The `AllowedMethods` option specifies nil-safe methods,
+      # in other words, it is a method that is allowed to skip safe navigation.
+      # Note that this is not an option to suppress pointing out this cop.
```

Comment:
I'm not sure what you mean by this last line?

Reply:
For other cop, the methods specified by `AllowedMethods` are suppressed (allowed) to be violated.
However, in this cop we want to express the difference of specifying a method that allows skip safe navigation.

Reply:
Changed wording. What do you think?